src/WebAdmin.cpp

```diff
@@ -341,6 +341,7 @@ void cWebAdmin::HandleWebadminRequest(cHTTPServerConnection & a_Connection, cHTT
 	if (ShouldWrapInTemplate)
 	{
 		cCSLock Lock(m_CS);
+		cLuaState::cLock lock(m_TemplateScript);
```

Comment:
I'm not too sure about the naming. It pretty easy to mix those two up @madmaxoft